rj_gameplay/stp/passing_cost.py

```diff
@@ -0,0 +1,54 @@
+import stp.rc as rc
+import numpy as np
+from typing import Optional
+import stp.role as role
+from math import atan2, pi
+
+
+class PassingCost(role.costFn):	
+	def __call__(robot: rc.Robot, prev_result: Optional["RoleResult"], world_state: rc.world_state) -> float:
+		pass_cost = 0
+		angle_threshold = 0.1
+		dist_threshold = 3
+		punish_amplifier = 1
+
+		our_robots = world_state.our_robots
+		their_robots = world_state.their_robots
+		recipient = robot
+		kicker = [robot for robot in our_robots if robot.has_ball_sense]
+		try:
+			kicker = kicker[0]
+		except:
+			print("Our team is not possessing the ball.")
```

Comment:
you should probably return a float('inf') if we don't posses the ball (this is a float which acts as infinity).